tasks/components/OrganizationMemberList.tsx

```diff
@@ -8,11 +8,11 @@ import { tw } from '@helpwave/common/twind/index'
 import type { Languages } from '@helpwave/common/hooks/useLanguage'
 import type { PropsWithLanguage } from '@helpwave/common/hooks/useTranslation'
 import { useTranslation } from '@helpwave/common/hooks/useTranslation'
-import Dropdown from '../icons/TriangleDown'
 import { Pagination } from './Pagination'
 import { Button } from './Button'
 import { Avatar } from './Avatar'
 import { Checkbox } from './user_input/Checkbox'
+import { ChevronDown as Dropdown } from 'lucide-react'
```

Comment:
We should avoid using `as`and should stick with the default naming of `lucide-react` to not confuse people in the future when they come across the icons.